File: Modul 2/Soal 04/PRAK204-NIM-Nama.c
#include <stdio.h>
#include <math.h>

int main(){
    float jr, t, v, l, k;

    printf("");
    scanf("%f %f", &jr, &t);

    v = 22.0 * jr * jr * t / 7.0;
    l = 2 * 22.0 * jr * (jr + t) / 7.0;
    k = 2 * 22.0 * jr / 7.0;
    
    printf("Volume = %.2f\n", v);
    printf("Luas = %.2f\n", l);
    printf("Keliling = %.2f\n", k);
    return 0;
}

File: Modul 2/Soal 04/PRAK204-NIM-Nama.py
inputs = []

while True:
    line = input()
    if not line:
        break
    inputs.extend(line.split())
inputs = list(map(float, inputs))

jr, t = inputs

v = 22 * jr * jr * t / 7
l = 2 * 22 * jr * (jr + t) / 7
k = 2 * 22 * jr / 7

print(f"Volume = {v:.2f}")
print(f"Luas = {l:.2f}")
print(f"Keliling = {k:.2f}")